integration/options/options-test.ts

```diff
@@ -0,0 +1,10710 @@
+import { protoMetadata } from './options';
+
+describe('options', () => {
+  it('generates types correctly', () => {
+    expect(protoMetadata).toMatchInlineSnapshot(`
```

Comment:
This inline snapshot is huge; can we move it to a regular non-inline snapshot?

Also, could you add one or two tests that specifically load options? I get if the snapshot is correct, it will work, but I'm mostly want test coverage of the API you're specifically using to look up options, so that they act as documentation / reminder of what not to break, if we happen to otherwise refactor some aspects of the metadata.

Reply:
I have extracted the snapshot to a separate file. 

I don't fully understand what you mean by the extra tests you are requesting?